frontend/view-edit-client/client-section.component.tsx

```diff
@@ -5,14 +5,30 @@ import AuditSummarySection from "./audit-summary-section.component";
 import { LastUpdate } from "./view-client.component";
 
 export default function ClientSection(props: ClientSectionProps) {
-  const [expanded, setExpanded] = React.useState(true);
+  const [expanded, setExpanded] = React.useState(
+    JSON.parse(
+      localStorage.getItem(`cu-client-section-expanded:${props.title}`)
+    ) === null
+      ? true
+      : JSON.parse(
+          localStorage.getItem(`cu-client-section-expanded:${props.title}`)
+        )
+  );
   const scope = useCss(css);
 
+  const toggleExpandAndStore = title => {
```

Comment:
nitpick -- no need to pass `title` into this function. Instead, you can just use `props.title` inside of the function